src/routes/courses.js

```diff
@@ -16,7 +16,11 @@ router.get('/:course_id', (req, res) => {
     res.json(courseloader.getCourseById(req.params.course_id));
   } else {
     res.status(404);
-    res.send('course_id does not exist!');
+    res.json({
+      status: 404,
+      message: 'Failed to find the course ID',
+      reason: 'Course ID does not exist',
```

Comment:
'reason' is not really needed, its content can be included in message as well, we want to make the message returned to frontend as clear as possible

Reply:
Sure.